src/components/InfoWindow/index.tsx

```diff
@@ -26,15 +27,17 @@ export default function InfoWindow() {
 
   return (
     <PopUpWindow id="popUpWindow" tabState={tabState} smoothLoopId={smoothLoopId}>
-      <Title>
-        <p className="CafeName">Honor</p>
-        <CancelIcon className="CancelIcon" />
-      </Title>
-      <Tab>
-        <div className="selected">업체제공사진</div>
-        <div className="">메뉴</div>
-        <div className="">인스타그램</div>
-      </Tab>
+      <Title.Wrapper>
+        <Title.Name>Honor</Title.Name>
+        <Title.Icon>
+          <CancelIcon />
+        </Title.Icon>
+      </Title.Wrapper>
+      <TabBar>
+        <Tab className="selected">업체제공사진</Tab>
+        <Tab className="">메뉴</Tab>
+        <Tab className="">인스타그램</Tab>
```

Comment:
이부분 Styled Component랑 클래스네임이 같이 쓰여있어서 뭔지 살짝 헷갈릴것 같은데
props로 selected로 전달해주는게 어떨까요??

```suggestion
        <Tab isSelected={true}>업체제공사진</Tab>
        <Tab>메뉴</Tab>
        <Tab>인스타그램</Tab>
```
이렇게 해주고, 실제 styled-component에서는 props로 받아와서 스타일 설정 해주면 좋을것같아요~

Reply:
```suggestion
        <Tab isSelected>업체제공사진</Tab>
        <Tab>메뉴</Tab>
        <Tab>인스타그램</Tab>
```

이것도 좋아보여요

Reply:
이게맞네요